gce/notebook/Dockerfile

```diff
@@ -8,9 +8,8 @@ RUN apt-get update \
 USER $NB_USER
 
 RUN conda install --yes -c defaults -c ioam -c bokeh/channel/dev -c intake \
-    bokeh=0.12.15dev1 \
+    bokeh \
```

Comment:
I recommend that we continue to pin to *something* just to be explicit, perhaps `0.12.16` ?